vendor/adeunis/pulse.yaml

```diff
@@ -9,7 +9,7 @@ hardwareVersions:
 # Firmware versions (at least one is mandatory)
 firmwareVersions:
   - # Firmware version
-    version: '1.0'
+    version: '2.01'
```

Comment:
do not delet old firmware version, this can cause problems to existing users, add the new firmware version only

Reply:
Under the Pulse sensor we have several version, we'll fix that next week

Thanks.